### ISSUE TITLE
StringObject escaping bug

### ISSUE THREAD

**Issue by rcannood** [viash-io/viash#113]
StringObjects aren't properly escaped.

`'` needs to be escaped in bash, r, python, and js scripts:
```
$ viash run src/test/resources/testbash/config.vsh.yaml -- README.md -s "foo'" --real_number 10 --whole_number 10
/home/rcannood/workspace/viash_temp//viash-run-testbash-jGiKfk: line 20: unexpected EOF while looking for matching `''
/home/rcannood/workspace/viash_temp//viash-run-testbash-jGiKfk: line 71: syntax error: unexpected end of file

$ viash run src/test/resources/testr/script.vsh.R -- README.md -s "foo'" --real_number 10 --whole_number 10
Error: unexpected string constant in:
"  "s" = 'foo'',
  "truth" = as.logical(toupper('"
Execution halted

$ viash run src/test/resources/testpython/ -- README.md -s "foo'" --real_number 10 --whole_number 10
code.py          config.vsh.yaml  tests/           
[rcannood@alya viash]$ viash run src/test/resources/testpython/config.vsh.yaml -- README.md -s "foo'" --real_number 10 --whole_number 10
  File "/home/rcannood/workspace/viash_temp//viash-run-testpython-Nh3MuY", line 8
    's': 'foo'',
              ^
SyntaxError: unterminated string literal (detected at line 8)

$ viash run src/test/resources/testjs/config.vsh.yaml -- README.md -s "foo'" --real_number 10 --whole_number 10
/home/rcannood/workspace/viash_temp/viash-run-testjs-MUXkZq:8
  's': 'foo'',
            ^^

SyntaxError: Invalid or unexpected token
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1032:15)
    at Module._compile (node:internal/modules/cjs/loader:1067:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

For csharp and scala, the same issue arises with `"`.
```
$ viash run src/test/resources/testcsharp/config.vsh.yaml -p docker -- README.md -s 'foo"' --real_number 10 --whole_number 10
/viash_automount/home/rcannood/workspace/viash_temp//viash-run-testcsharp-nGjIOk(13,12): error CS1003: Syntax error, ',' expected
/viash_automount/home/rcannood/workspace/viash_temp//viash-run-testcsharp-nGjIOk(13,12): error CS1010: Newline in constant
/viash_automount/home/rcannood/workspace/viash_temp//viash-run-testcsharp-nGjIOk(13,12): error CS0746: Invalid anonymous type member declarator. Anonymous type members must be declared with a member assignment, simple name or member access.
/viash_automount/home/rcannood/workspace/viash_temp//viash-run-testcsharp-nGjIOk(13,14): error CS1003: Syntax error, ',' expected

$ viash run src/test/resources/testscala/config.vsh.yaml -- README.md -s 'foo"' --real_number 10 --whole_number 10
/home/rcannood/workspace/viash_temp/viash-run-testscala-rryV6i:20: error: unclosed string literal
  "foo"",
       ^
/home/rcannood/workspace/viash_temp/viash-run-testscala-rryV6i:25: error: ';' expected but ',' found.
  Some("The default value."),
```

**Comment by rcannood** [comment]
Created branch `fix/issue_113`. 

- [ ] We should create a file with all 'normal' characters that should be passable to viash components. Unit tests can read in this file and pass it to the underlying component. 
- [ ] Fix components accordingly

**Comment by rcannood** [comment]
Solved in #240!
